front-end/src/app/shared/components/inputs/amount-input/amount-input.component.ts

```diff
@@ -1,33 +1,28 @@
 import { ChangeDetectorRef, Component, Input, OnChanges, OnInit, ViewChild } from '@angular/core';
-import { FormControl, Validators } from '@angular/forms';
 import { Store } from '@ngrx/store';
-import { F3xSummary } from 'app/shared/models/f3x-summary.model';
-import { selectActiveReport } from 'app/store/active-report.selectors';
 import { InputNumber } from 'primeng/inputnumber';
-import { takeUntil } from 'rxjs';
 import { BaseInputComponent } from '../base-input.component';
+import { Transaction } from 'app/shared/models/transaction.model';
 
 @Component({
   selector: 'app-amount-input',
   styleUrls: ['./amount-input.component.scss'],
   templateUrl: './amount-input.component.html',
 })
 export class AmountInputComponent extends BaseInputComponent implements OnInit, OnChanges {
-  @Input() memoCodeReadOnly = false;
   @Input() contributionAmountReadOnly = false;
-  @Input() memoItemHelpText =
-    'The dollar amount in a memo item is not incorporated into the total figures for the schedule.';
   @Input() negativeAmountValueOnly = false;
   @Input() showAggregate = true;
 
+  @Input() memoCodeReadOnly = false;
+  @Input() memoItemHelpText =
```

Comment:
it seems like we don't ever assign this input from above.  Considering that and the duplicate definition of the default value, here and in the memo-input, maybe we can remove it from the amount-input component

Reply:
I hate to strip out capacity for customization, but it would be very awkward to have these values in the memo code component being set by the default values in the amount input component.

For now, I've rejiggled this so that the Memo Code Component has the only default values, and you may or may not pass in overrides from above.

```
@Input() overrideMemoItemHelpText: string | undefined;
@Input() overrideMemoCodeReadOnly: boolean | undefined;

memoItemHelpText = 'The dollar amount in a memo item is not incorporated into the total figures for the schedule.';
memoCodeReadOnly = false;
```

and then later

```
if (this.overrideMemoCodeReadOnly) this.memoCodeReadOnly = this.overrideMemoCodeReadOnly;
if (this.overrideMemoItemHelpText) this.memoItemHelpText = this.overrideMemoItemHelpText;
```

Reply:
:+1: